com.ranfdev.Geopard.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "com.ranfdev.Geopard",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "geopard",
+    "tags" : [
+        "devel"
+    ],
+    "finish-args" : [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
```

Comment:
this is not allowed, so it will fail. you need to use flatpak-builder-tools to generate a manifest from the `Cargo.lock`

Reply:
I've bundled everything inside an archive now